test_regress/t/t_vpi_cbs_called.cpp

```diff
@@ -55,6 +55,7 @@ bool verbose = false;
 #define CHECK_RESULT_NZ(got) \
     if (!(got)) { \
         printf("%%Error: %s:%d: GOT = NULL  EXP = !NULL\n", __FILE__, __LINE__); \
+        Verilated::gotFinish(true); \
```

Comment:
Better to stop

vl_stop(__FILE__, __LINE__, "check_result");

Or, set an error flag you check at the end.  Setting finish is generally considered "good".

Thanks.